calicoctl/commands/node/diags.go

```diff
@@ -169,7 +212,7 @@ func writeDiags(cmds diagCmd, dir string) {
 
 	content, err := command.Output()
 	if err != nil {
-		fmt.Println(err)
```

Comment:
We're losing the error in this log the way it's written now.  I think we need to keep that so we can tell what failed.

Reply:
I'll get this setup so we see the returned error from the command,. Before we were only getting an unexplained `exit status 1` here.